src/tools/rbd/action/Bench.cc

```diff
@@ -389,7 +389,7 @@ int do_bench(librbd::Image& image, io_type_t io_type,
       std::cout.width(5);
       std::cout << (int)elapsed.count();
       std::cout.width(10);
-      std::cout << (int)(ios - io_threads);
+      std::cout << (uint64_t)(ios - io_threads);
```

Comment:
I don't think the cast is necessary here.  With your change the type of both `ios` and `io_threads` is uint64_t.

Reply:
updated